src/go/install.sh

```diff
@@ -270,6 +270,8 @@ if [[ "${TARGET_GO_VERSION}" != "none" ]] && [[ "$(go version 2>/dev/null)" != *
     fi
     curl -fsSL -o /tmp/go.tar.gz.asc "https://golang.org/dl/go${TARGET_GO_VERSION}.linux-${architecture}.tar.gz.asc"
     gpg --verify /tmp/go.tar.gz.asc /tmp/go.tar.gz
+    echo "Removing existing Go ${TARGET_GOROOT}..."
+    rm -rf $TARGET_GOROOT/*
```

Comment:
```suggestion
    echo "Removing existing Go ${TARGET_GOROOT}..."
    rm -rf "${TARGETG_GOROOT:?}/"*
```

To prevent accidental deletion if `TARGET_GOROOT` is empty or unset.